client/src/node/extension.ts

```diff
@@ -151,11 +151,20 @@ function triggerProfileUpdate(): void {
   const activeProfileName = profileConfig.getActiveProfile();
   if (activeProfileName in profileList || activeProfileName === "") {
     updateStatusBarProfile(activeProfileStatusBarIcon);
-    commands.executeCommand(
-      "setContext",
-      "SAS.connectionType",
-      profileList[activeProfileName].connectionType || ConnectionType.Rest
-    );
+
+    const connectionType =
+      profileList[activeProfileName].connectionType || ConnectionType.Rest;
+
+    //Set the connection type
+    commands.executeCommand("setContext", "SAS.connectionType", connectionType);
+    if (connectionType === ConnectionType.Rest) {
+      //Check to see if we are directly connected (Ie. we have a serverId)
+      if ("serverId" in profileList[activeProfileName]) {
+        commands.executeCommand("setContext", "SAS.connection.direct", true);
+      }
+    } else {
+      commands.executeCommand("setContext", "SAS.connection.direct", false);
+    }
```

Comment:
I think this could probably be simplified to something like...
```js
commands.executeCommand(
  'setContext',
  'SAS.connection.direct',
  connectionType === ConnectionType.Rest && !!profileList[activeProfileName]?.serverId
)
```

but this seems like a simple enough change to incorporate. I was imagining something more complicated 😁. We will have to talk to Sam/Cristina/Becky to see how this changes messaging on the welcome screen, but that needs to be adjusted now anyway since it reads...

> Your connection does not support SAS content navigation within the SAS Extension for Visual Studio Code. You can access files only using the Explorer view.